### ISSUE TITLE
ClassNotFoundException in Logger.php line 23

### ISSUE THREAD

**Issue by CodyBohn** [silexphp/Silex-Skeleton#60]
Currently, running "composer create-project fabpot/silex-skeleton skeleton" yields;

**ClassNotFoundException in Logger.php line 23:**

**Attempted to load interface "LoggerInterface" from namespace "Symfony\Component\HttpKernel\Log".
Did you forget a "use" statement for "Psr\Log\LoggerInterface"?**

Logger implements LoggerInterface and DebugLoggerInterface from Symfony\Component\HttpKernel

symfony/http-kernel (v3.0.0) is installed by default and does not contain LoggerInterface. Explicitly requiring symfony/http-kernel (v2.8.0) resolves the issue.

**Comment by nickstylez** [comment]
@CodyBohn Add ```"symfony/http-kernel": "^2.8"``` to your composer.json in the root folder. The current version of this skeleton uses http-kernel version 3.x which doesn't work (the "LoggerInterface" was deprecated in 2.8).

**Comment by CodyBohn** [comment]
@nickstylez That works as I mentioned in the last line. I opened the issue because it will hinder any install without making that change manually.

**Comment by vikbert** [comment]
@nickstylez  thanks, it works for me.

**Comment by piotr-mroczek** [comment]
@reynke it work to me too. thx

**Comment by ashorlivs** [comment]
@reynke thanks !

**Comment by tstrijdhorst** [comment]
Thanks, this should be updated I guess? Bit weird to have it broken out of the box.

**Comment by TakTaki** [comment]
@reynke Thanks, can't believe this issue is was open 4 months ago...

**Comment by michelerota** [comment]
+1

**Comment by xstable** [comment]
+1

Please fix it for in the composer-file. 
I'm a newbie with silex. this are my first experiences with. Would be nice, if the skeleton would work out ofthe box to keep the first steps simple as possible!

After fixing this, I got another error: 
```
ContextErrorException in Logger.php line 311:
Warning: date_default_timezone_get()
```
But for this I'll open an new Issue...

**Comment by fabpot** [comment]
fixed in 0f377f7

**Comment by Germain-Italic** [comment]
Thanks @fabpot
